src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/RazorSemanticTokensEndpoint.cs

```diff
@@ -151,11 +151,6 @@ private async Task<DocumentSnapshot> TryGetCodeDocumentAsync(string absolutePath
                 return documentSnapshot;
             }, cancellationToken, TaskCreationOptions.None, _foregroundDispatcher.ForegroundScheduler);
 
-            if (document is null)
-            {
-                return null;
```

Comment:
If it's null should we be doing a Debug.Fail equivalent?

Reply:
Hmm maybe, but this was more of "'if x is null return null else return x' is very silly" than it was an attempt to change behavior.